src/components/ExerciseCard.jsx

```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link';
+
+import useStore from '../hooks/useStore';
+
+import PlayIcon from './Icons/PlayIcon';
+import StyledCards from './StyledCards';
+
+export default function ExerciseCard() {
+	const exerciseTimes = useStore(state => state.exerciseTimes);
+	const length = exerciseTimes.length;
+	const lastTime = exerciseTimes[length - 1];
+	return (
+		<Link href="/exercise">
+			<StyledCards variant="exercise">
```

Comment:
I think it would be better to use singular here 🤔 I see that you have different variants but in the end you only render one card.
```suggestion
			<StyledCard variant="exercise">
```